src/main/java/core/basesyntax/fruitservice/FruitStorageService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.fruitservice;
+
+public class FruitStorageService {
```

Comment:
I do not know why do you need to divide this class from your class with storage, but I think it's optional